packages/plugins/python-docutils/python-docutils.spec

```diff
@@ -0,0 +1,80 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Created by pyp2rpm-3.3.8
+%global pypi_name docutils
+
+Name:           python-%{pypi_name}
+Version:        0.19
+Release:        1%{?dist}
+Summary:        Docutils -- Python Documentation Utilities
+
+License:        public domain, Python, 2-Clause BSD, GPL 3 (see COPYING.txt)
+URL:            https://docutils.sourceforge.io/
+Source0:        https://files.pythonhosted.org/packages/source/d/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+
+
+%description
+%{summary}
+
+
+%package -n     python%{python3_pkgversion}-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{pypi_name}}
+Requires:       python%{python3_pkgversion}-setuptools
+
+
+%description -n python%{python3_pkgversion}-%{pypi_name}
+%{summary}
+
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+set -ex
+%autosetup -n %{pypi_name}-%{version}
+# Remove bundled egg-info
+rm -rf %{pypi_name}.egg-info
+%{?scl:EOF}
+
+
+%build
+%{?scl:scl enable %{scl} - << \EOF}
+set -ex
+%py3_build
+%{?scl:EOF}
+
+
+%install
+%{?scl:scl enable %{scl} - << \EOF}
+set -ex
+%py3_install
+%{?scl:EOF}
+
+
+%files -n python%{python3_pkgversion}-%{pypi_name}
+%license COPYING.txt
+%doc README.txt docutils/parsers/rst/include/README.txt docutils/writers/s5_html/themes/README.txt test/functional/README.txt test/functional/output/README.txt tools/dev/README.txt tools/editors/README.txt tools/editors/emacs/README.txt tools/editors/emacs/tests/README.txt
+%{_bindir}/docutils
+%{_bindir}/rst2html.py
+%{_bindir}/rst2html4.py
+%{_bindir}/rst2html5.py
+%{_bindir}/rst2latex.py
+%{_bindir}/rst2man.py
+%{_bindir}/rst2odt.py
+%{_bindir}/rst2odt_prepstyles.py
+%{_bindir}/rst2pseudoxml.py
+%{_bindir}/rst2s5.py
+%{_bindir}/rst2xetex.py
+%{_bindir}/rst2xml.py
+%{_bindir}/rstpep2html.py
```

Comment:
this will (possibly) clash with the binaries from `python3-docutils` that comes from the BaseOS, and as `python-daemon` doesn't use any of these, I'd prefer to `%exclude them.